### ISSUE TITLE
Repeated code

### ISSUE THREAD

**Issue by oturnermajor** [Kata-Martagon/Stopwatch-NogaMatt#17]
https://github.com/Kata-Martagon/Stopwatch-NogaMatt/blob/master/particles.js#L20-L60

I feel like there must be a better way to do this. You're setting variables with the same name to different values 4 times (in functional programming it's best not to mutate data), making an object using a constructor (unnecessary when all they do is return an object with the values given), and then pushing that object into an array. 

The function is called very soon after the empty array is initialised... why not just initialise the array with the 4 objects already in it with the values that you need? Like: 

```javascript
var midX = c.width / 2;
var midY = c.height / 2;
var totalParticles = 0;
var origins = [{
  // Hundredths
  mass: 1000,
  position: {x: midX, y: midY - 250}, // no need for Vector constructor either
  velocity: {x: 1.45 y:0},
  colour: 'rgb(122,214,245)',
  scale: 1,
  spread: Math.PI / 10,
},
...
]
```

Feel free to disregard if there's a good reason for doing it the way you currently are, I have only skimmed the code so I could be missing something.

**Comment by matthewiiv** [comment]
Good point. I am creating multiple objects and pushing them into an array, but I was unsure how to create multiple things with lots of properties properly. Just went with the simple solution of using the same variables multiple times. 
Will look to refactor in the way suggested.
